Userland/Libraries/LibCore/System.cpp

```diff
@@ -1655,4 +1656,39 @@ ErrorOr<void> posix_fallocate(int fd, off_t offset, off_t length)
 }
 #endif
 
+// This constant is copied from LibFileSystem. We cannot use or even include it directly,
+// because that would cause a dependency of LibCore on LibFileSystem, effectly rendering
+// the distinction between these libraries moot.
+static constexpr StringView INTERNAL_DEFAULT_PATH_SV = "/usr/local/sbin:/usr/local/bin:/usr/bin:/bin"sv;
```

Comment:
We don't have LibFileSystem available in LibCore, and including a header risks breaking something at runtime. I'd rather copy a constant over than cause a weird Heisenbug that needs to be figured out later.

Yes, I could just use the literal in the function, but I'd prefer naming these magic values, especially since it's a copy of somewhere else.

Reply:
i meant you could remove this constant altogether from here and settle for the following signature:
```cpp
ErrorOr<String> resolve_executable_from_environment(StringView filename, int flags, default_path)
```
and feed it in call-site instead.

Reply:
Why would someone want to change the default_path? I don't see any advantage, and using any other value will introduce corner cases and subtle differences in behavior.

No, I'd rather keep the signature the same.